### ISSUE TITLE
Improve information on response matrix construction (CustomInstrument)

### ISSUE THREAD

**Issue by sguillot** [xpsi-group/xpsi#138]
Help the user figure out how to solve the ResponsError:
```
Each row and column of the matrix must contain at least one positive number.
Try reducing the \"gap\" between the minimum input energy 
```

**Comment by thjsal** [comment]
Is the suggestion to add this peace of text to the code (where?), or do you mean more information than this is needed?

**Comment by sguillot** [comment]
Probably both in the code and the documentation.
Also, this should be solved with #194 and #320

**Comment by lmauviard** [comment]
Closed by the PR #424
